### DISCUSSION TITLE
Why not using docker-sync or mutagen?

### DISCUSSION THREAD

**Discussion by Axel29** [markshust/docker-magento#368] [Q&A] [answered]
Hello,

I was wondering why you were not using Mutagen nor Docker-sync?
I've recently used Magento Cloud's official Docker instance and it is used and quite performant.

Thank you,
Axel

**Comment by markshust** [comment, marked as answer]
I used docker-sync many years ago, and had issues with reliability. There were times files & folders randomly stopped syncing. To me, reliability is much more important that performance, so the tradeoff was made to use the selective sync method instead.

I was one of the first adopters of Mutagen, and similarly to docker-sync, had issues with reliability of syncing.

I may take an additional look at docker-sync with NFS, however the inotify events must be sent in order to do local development without pain. It's been a while since I've tested either of these solutions, so it may be worth an additional look. I've been through the ringer though with all of these solutions, and the current solution has by-far been the best solution in terms of reliability vs performance.

**Reply by Axel29** [reply]
Thank you for such a detailed answer, I couldn't hope for more :) 
I totally understand your point of view and agree with you, I'm just new to Docker and I've noticed that difference with magento cloud"s official docker so I was curious :)

Thank you agin!
